### PR TITLE
Remove superfluous toot length check

### PR DESCRIPTION
**Pull request by noiob**
Also fix the check that was ignored before

### CODE REVIEW

File: app/javascript/flavours/glitch/features/compose/components/compose_form.js [hunk 1]
```diff
@@ -94,7 +94,7 @@ class ComposeForm extends ImmutablePureComponent {
     const { isSubmitting, isChangingUpload, isUploading, anyMedia } = this.props;
     const fulltext = this.getFulltextForCharacterCounting();
 
-    return !(isSubmitting || isUploading || isChangingUpload || length(fulltext) > maxChars || (!fulltext.trim().length && !anyMedia));
+    return !(isSubmitting || isUploading || isChangingUpload || length(fulltext) >= maxChars || (!fulltext.trim().length && !anyMedia));
```
Comment (ClearlyClaire) [inline app/javascript/flavours/glitch/features/compose/components/compose_form.js:97]: Why this change? If the full text length has reached exactly the max chars, it should be still considered acceptable, shouldn't it?

**Review by ClearlyClaire — CHANGES_REQUESTED** [review]

File: app/javascript/flavours/glitch/features/compose/components/compose_form.js (hunk 1 above)
Comment (noiob) [reply app/javascript/flavours/glitch/features/compose/components/compose_form.js:97]: yeah, you're right

**Review by ClearlyClaire — APPROVED** [review]
